Homework_8/Task_1.py

```diff
@@ -0,0 +1,13 @@
+# პროგრამამ უნდა მოგვთხოვოს სტრიქონის შეყვანა.
+# დაბეჭდოს შეყვანილი სტრიქონის ყველა ლუწი ინდექსის მქონე სიმბოლო, გარდა "e"- სიმბოლოსი.
+
+user_input = input("please  enter text: ")
+new_word = ""
+
+for i in range(len(user_input)):
```

Comment:
პირდაპირ მხოლოდ ლუწ ინდექსებს გადაყოლოდი შეგეძლო for i in range(0, len(user_input), 2)-ით